server/test/sc/server/client/PlayerListener.kt

```diff
@@ -0,0 +1,14 @@
+package sc.server.client
+
+import sc.api.plugins.host.IPlayerListener
+import sc.protocol.responses.ProtocolMessage
+
+class PlayerListener : IPlayerListener {
+    var playerEventReceived = false
```

Comment:
this variable seems superfluous

Reply:
It is being used by [server/test/sc/protocol/requests/RequestTest.kt](https://github.com/CAU-Kiel-Tech-Inf/backend/blob/refactor/convert-RoundBasedGameInstance/server/test/sc/protocol/requests/RequestTest.kt)

Reply:
yes, but the value of that variable can probably be inferred from the other variable in this class, right?

Reply:
could be.
This PR aims to convert `RoundBasedGameInstance`, though.
Checking for and cleaning up possibly redundant variables in test classes just doesn't belong here.

Reply:
fair ^^

Reply:
Added new issue for that #353